src/seedu/addressbook/Main.java

```diff
@@ -72,6 +72,7 @@ private void start(String[] launchArgs) {
 
     /** Prints the Goodbye message and exits. */
     private void exit() {
+    	ui.showTaggingList(this.addressBook.getTaggingList()); 
```

Comment:
Indentation problem here. Our coding standard requires you to use 4 spaces instead of tabs. You can configure Eclipse to convert tabs to spaces automatically.